src/controllers/controller.line.js

```diff
@@ -170,43 +169,6 @@ module.exports = DatasetController.extend({
 		return values;
 	},
 
-	calculatePointY: function(value, index, datasetIndex) {
```

Comment:
Add the removal of this method to the migration guide

Reply:
done